wgengine/userspace.go

```diff
@@ -1005,6 +1005,7 @@ func (e *userspaceEngine) getStatus() (*Status, error) {
 	closing := e.closing
 	peerKeys := make([]key.NodePublic, len(e.peerSequence))
 	copy(peerKeys, e.peerSequence)
+	localAddrs := append([]tailcfg.Endpoint(nil), e.endpoints...)
```

Comment:
https://pkg.go.dev/golang.org/x/exp/slices#Clone is a bit prettier these days

localAddrs := slices.Clone(e.endpoints)